projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/search-ai-results/search-ai-results.component.scss

```diff
@@ -98,7 +141,7 @@
             padding-left: 8px;
 
             &-header {
-              margin-top: 16px;
+              margin-top: 24px;
```

Comment:
On Figma I can see that space between header and divider is 12px, I believe now it is greater. Could you please correct? 
<img width="819" alt="image" src="https://github.com/user-attachments/assets/ac415652-0b53-4a5d-8c5a-a80a30bbbda8">

Reply:
Actually I can see there is 16px (I think there is some little space added by browser so you need to use different value of margin to achieve 12px).